cli/README.md

```diff
@@ -1,55 +1,74 @@
 
-## Configuring a project
+## Initializing a Project
 
-In ordert to configure the CLI in a project you'll run `xata init --db=https://{workspace}.xata.sh/db/{database}`. The flag `--db` is not mandatory but recommended, because this way you won't be asked for the workspace/database and thus, a global API key is not required.
+To initialize a new database, run `xata init` in your project's directory.
 
-This command will ask you a few questions and will run a few operations:
+- if you don't have a [workspace](https://docs.xata.io/concepts/workspaces), you will be prompted to create one.
+- if you have workspaces, you'll be asked to choose one, or to create a new one.
 
-- It will create a project configuration file (`.xatarc`).
-- It will create or update your `.env` file to store a database-scoped API key. (At the time of writing these docs it'll actually store a personal API key, but this will change soon).
-- It will ask you if you want to install the SDK and/or use the TypeScript/JavaScript code generator.
+A workspace is a logical grouping of databases, usually analogous to an organization or team, so this is the first step. Once you've chosen a workspace, you will be given the option to either create a new one, or use an existing one for your project.
 
-## Code generation
+After choosing a workspace and a database, you're ready to go: the CLI will walk you through next steps, ultimately leading you to code generation.
 
-You can use the Xata SDK (`@xata.io/client`) directly or with code generation. The benefit of using code generation is that you'll have a Xata client with specific types and classes for your database schema. This is great for TypeScript because it ensures type safety, but also for JavaScript because you can benefit from its type definitions if you use development tools such as IDEs or text editors that support them.
+## Code Generation
 
-Code generation will be executed automatically in some cases by the CLI (such as when creating a new Xata branch). You can always invoke it directly with `xata codegen`.
+We strongly recommend using the CLI to generate your Xata fetcher (sometimes called a _client_) for you. This will help make your queries to Xata predictable and safe, while also providing autocomplete and safety.
 
-If you make changes to the schema through the UI or the REST API, you will need to run `xata codegen` to update your local generated code with the new changes.
+Once you have a generated fetcher, you can use import it into your project and using it. The `.` character (object property access in JavaScript) will provide autocompletion hints. You can now also use the [SDK Playground](https://docs.xata.io/sdk/playground) in the Xata web interface to test out your queries and then copy and paste them into your applications.
```

Comment:
> you can use import it into your project and using it. The `.` character (object property access in JavaScript) will provide autocompletion hints

This part is confusing.

Reply:
Thanks a lot! Will think of a solution. 🧠 ⏳

Reply:
Done in https://github.com/xataio/client-ts/pull/400/commits/1e8f128da51b08b1a9b8598c4fe2baf2f857a4f6